### ISSUE TITLE
[Fefature/Enhancement Request] Allow configuration of max upload size via Lua and NGINX config.

### ISSUE THREAD

**Issue by teward** [danbooru/danbooru#5778]
There is no mechanism to edit nginx.conf if you change the max_upload_size.

There is the mechanism with Lua scripts, etc. to get data values out of the configured environment vars in .env.local.  That (or extracting from the ruby config) would make it easier to make changes and not have git deltas that are substantial for just nginx.conf (since the default danbooru configs can be overridden with env vars).

**Comment by ghost** [comment]
This might not be possible—the `client_max_body_size` directive does not seem to accept a variable (and according to the nginx ticket, the body size is validated before Lua runs, which means it wouldn't work anyway).

https://groups.google.com/g/openresty-en/c/qgSjFSSDVrg?pli=1
https://trac.nginx.org/nginx/ticket/586#no1

**Comment by evazion** [comment]
Yeah, I attempted this before and couldn't find a way to get it to work.
